src/tests/nativeaot/SmokeTests/SharedLibrary/SharedLibrary.csproj

```diff
@@ -5,26 +5,18 @@
     <CLRTestPriority>0</CLRTestPriority>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NativeLib>Shared</NativeLib>
-<<<<<<< HEAD
 
     <WasiReactorTestUsingNodeJS Condition="'$(TargetOS)' == 'wasi'">true</WasiReactorTestUsingNodeJS>
 
-    <!-- Unable to compile a project with the Library output type for apple mobile devices -->
-    <CLRTestTargetUnsupported Condition="'$(TargetsAppleMobile)' == 'true'">true</CLRTestTargetUnsupported>
-
-    <!-- https://github.com/dotnet/runtimelab/issues/2648 -->
-    <CLRTestTargetUnsupported Condition="'$(TargetOS)' == 'wasi'">true</CLRTestTargetUnsupported>
-=======
     <BuildAsStandalone>false</BuildAsStandalone>
-    <!-- Unable to compile a project with the Library output type for apple mobile devices -->
-    <CLRTestTargetUnsupported Condition="'$(TargetsAppleMobile)' == 'true'">true</CLRTestTargetUnsupported>
+    <!-- Unable to compile a project with the Library output type for apple mobile devices or WASI -->
+    <CLRTestTargetUnsupported Condition="'$(TargetsAppleMobile)' == 'true' or '$(TargetOS)' == 'wasi'">true</CLRTestTargetUnsupported>
```

Comment:
This should stay as a distinct condition pointing to `https://github.com/dotnet/runtimelab/issues/2648`.

Reply:
THanks, missed that.